### DISCUSSION TITLE
[✨] Make `QwikIntrinsicElements` extend `IntrinsicSVGElements`, or otherwise expose SVG element types

### DISCUSSION THREAD

**Discussion by caass** [QwikDev/qwik-evolution#158] [Proposals For Qwik]
### Is your feature request related to a problem?

I'm trying to extend the `<svg>` builtin, and I'd like to be able to accept the same props and forward them on. I can't seem to make it happen, however, because [`QwikIntrinsicElements`](https://github.com/BuilderIO/qwik/blob/73ef1f9b6a2d7bb69d7f9fa4f2a360e69ab99339/packages/qwik/src/core/render/jsx/types/jsx-qwik-elements.ts#L23) doesn't extend [`IntrinsicElements`](https://github.com/BuilderIO/qwik/blob/73ef1f9b6a2d7bb69d7f9fa4f2a360e69ab99339/packages/qwik/src/core/render/jsx/types/jsx-generated.ts#L1168) but rather [`IntrinsicHtmlElements`](https://github.com/BuilderIO/qwik/blob/73ef1f9b6a2d7bb69d7f9fa4f2a360e69ab99339/packages/qwik/src/core/render/jsx/types/jsx-qwik-elements.ts#L23) I don't have access to those types.

### Describe the solution you'd like

Modify the `QwikIntrinsicElements` type to extend `IntrinsicElements`

### Describe alternatives you've considered

I hadn't really considered any -- I guess another alternative is to expose `IntrinsicSVGElements` directly.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
